src/style.css

```diff
@@ -1 +1,84 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  background-color: #e5e5e5;
+}
+
+.container {
+  padding: 0;
+  margin: 0;
+  position: relative;
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+  max-width: 100%;
+  min-width: 1024px;
+  background-color: #fff;
```

Comment:
Please pay attention to your CSS properties order
Please fix all cases
https://9elements.com/css-rule-order/